IBPSA/BoundaryConditions/Validation/IsotropicAndPerezDiffuseRadiation.mo

```diff
@@ -0,0 +1,94 @@
+within IBPSA.BoundaryConditions.Validation;
+model IsotropicAndPerezDiffuseRadiation
+  "Partial model to run BESTEST validation case studies for weather data processing"
+  extends
+    IBPSA.BoundaryConditions.SolarIrradiation.BaseClasses.PartialSolarIrradiation;
+  Modelica.Blocks.Interfaces.RealOutput HPer(
+     final quantity="RadiantEnergyFluenceRate",
+     final unit="W/m2") "Radiation per unit area"
```

Comment:
It would be beneficial if the description contains the difference to `H`. That it is the irradiance according to Perez or something similar.

Reply:
As above, will adress this weekend